drv_helpers.c

```diff
@@ -522,11 +522,14 @@ void *drv_dumb_bo_map(struct bo *bo, struct vma *vma, uint32_t map_flags)
 
 int drv_bo_munmap(struct bo *bo, struct vma *vma)
 {
+	int ret = 0;
```

Comment:
0 seems to be a success case for [munmap](https://www.man7.org/linux/man-pages/man3/munmap.3p.html) 
Should the default value be -1 ?